registration/views.py

```diff
@@ -549,6 +561,20 @@ def get_context_data(self, form, **kwargs):
         d['postal_code'] = self.request.session[SESSION_POSTAL_CODE]
         d['email'] = self.request.session[SESSION_EMAIL]
 
+        form_test = self.form_list[unicode(self.steps.current)]
+        if 'license_id' in form_test.base_fields:
+            if SESSION_LICENSE_ID_FORMATS in self.request.session:
+                license_id_formats = [str(license_id_format) for license_id_format in
```

Comment:
@waseem-omar the reason why I'm doing this is to strip out the unicode `u` from the items list  so I'm looping over them to eliminate them.Secondly I'm dumping the list as a JSON because this list will be in hidden div and the content of that div is manipulated by  JS , we want to  parse that content as a  list in JS , if I  omit the json conversion on dajngo side , the list content will be  processed as a normal string.

@danaspiegel  I will pull out the conversion to a separate method